site/docs/contribute.md

```diff
@@ -45,6 +45,17 @@ The Iceberg community prefers to receive contributions as [Github pull requests]
 * If a PR is related to an issue, adding `Closes #1234` in the PR description will automatically close the issue and helps keep the project clean
 * If a PR is posted for visibility and isn't necessarily ready for review or merging, be sure to convert the PR to a draft
 
+### Merging Pull Requests
+
+Most pull requests can be merged once a single [committer](https://www.apache.org/foundation/how-it-works/#committers) other than the author is satisfied with the code in the PR. Before merging all review comments should be addressed either by making changes or agreeing the request is out of scope for the PR. For additions to public APIs committers should wait at least 24 hours before merging to ensure there is no additional feedback from members of the community. [Committers are trusted](https://infra.apache.org/new-committers-guide.html#the-committers-way) to act in the best [interest of the project](https://community.apache.org/projectIndependence.html#apache-projects-are-managed-independently). If a committer feels there might be a conflict of interest with a pull request they review, they are encouraged to ask for another committer to review the pull request. 
```

Comment:
Shall we give a few concrete examples on what it means to have a conflict of interest while reviewing a PR?

Reply:
I'm a bit late to this, but I also had the same question as Anton. In the end, maintainers have to use their best judgement but if we're mentioning "conflict of interest" in the guidelines, I think we should give a few examples to make sure there's a shared understanding of what that can mean.

Reply:
Added a couple of sentences, please let me know if this is sufficient